sql/upgrade/sl3.0.sql

```diff
@@ -1054,7 +1069,7 @@ BEGIN
 END;
 $$ LANGUAGE PLPGSQL;
 
-SELECT payment_migrate(p.id, p.trans_id, cast(p.exchangerate as numeric), p.paymentmethod_id)
+SELECT pg_temp.payment_migrate(p.id, p.trans_id, cast(p.exchangerate as numeric), p.paymentmethod_id)
```

Comment:
Has nothing to do with performance fixes in the migration. But nice catch. I would have put it in its own commit because it's thematically different from the other changes.

Reply:
This specific fix has nothing to do but payment_migrate is the 3rd longuest query, accounting for almost 10% of migration time.